test/unit/BondingManager.js

```diff
@@ -1,6 +1,6 @@
 import Fixture from "./helpers/Fixture"
-import expectThrow from "../helpers/expectThrow"
 import expectRevertWithReason from "../helpers/expectFail"
+import expectThrow from "../helpers/expectThrow"
```

Comment:
Since we're still keeping this import we can just move it back to its original location above L2 to reduce the size of the diff.

Reply:
Yeah I had deleted this first, but then `public view` comments weren't returning revert reasons so I added it again, will fix the position

Reply:
fixed in 2eb6a0f